src/en/howto-windows.md

```diff
@@ -0,0 +1,85 @@
+## This guide is currently experimental and under dev
+
+Juju is capable of orchestrating Windows workloads thanks to the enablement
+efforts of [CloudBase Solutions](http://cloudbaseit.com).
+
+> Getting started with orchestrating windows workloads today is still a work in
+progress - as it's only available on 2 substrates. MAAS 1.8 experimental, and
+OpenStack with a custom image loaded in Cinder.
```

Comment:
Actually, this will work on any cloud provider that allows you to upload your own image, and for which we have support in cloudbase-init:

https://github.com/stackforge/cloudbase-init/tree/master/cloudbaseinit/metadata/services